tests/views/test_update_event_date.py

```diff
@@ -70,10 +71,14 @@ def setup_data(self):
         )
 
     @requests_mock.mock()
-    def test_update_event_date_view(self, mock_request):
+    @patch(
```

Comment:
Mock requests are causing issues between tests, because of repetition between classes it wasn't showing up. As soon as I started refactoring test_collection_exercise.py I saw issues. It isn't connected to the decorator as the patches are targetted to just the collection_exercise.py view